src/index.js

```diff
@@ -1,25 +1,139 @@
+    const selectInput = document.querySelectorAll('.input-div');
+    for (let i = 0; i < getting.length; i += 1) {
+      if (selectInput[i].classList.contains('changeBg')) {
+        getting[i].completed = true;
+      } else {
+        getting[i].completed = false;
+      }
+      empty.push(getting[i]);
+      localStorage.setItem('list', JSON.stringify(empty));
+    }
+  });
+  const clearAll = document.querySelector('#clear');
+  clearAll.addEventListener('click', () => {
+    const getting = JSON.parse(localStorage.getItem('list'));
+    const variable = document.querySelectorAll('.changeBg');
+    for (let i = 0; i < variable.length; i += 1) {
+      form.removeChild(variable[i]);
+    }
+    const empty = [];
+    for (let i = 0; i < getting.length; i += 1) {
+      if (getting[i].completed === true) {
+            continue; // eslint-disable-line
+      }
+      empty.push(getting[i]);
+    }
+    localStorage.setItem('list', JSON.stringify(empty));
+  });
 
-        `;
+  deleteBtn.addEventListener('click', () => {
+    form.removeChild(list);
+    const getFromLocalStorage = JSON.parse(localStorage.getItem('list'));
+    const result = getFromLocalStorage.filter((word) => word.description === listText.textContent);
+    const empty = [];
+    for (let i = 0; i < getFromLocalStorage.length; i += 1) {
+      if (result[0].description === getFromLocalStorage[i].description) {
+            continue; // eslint-disable-line
+      }
+      empty.push(getFromLocalStorage[i]);
+    }
+    localStorage.setItem('list', JSON.stringify(empty));
+  });
+
+  editBtn.addEventListener('click', () => {
+    const editInput = document.createElement('input');
+    editInput.type = 'text';
+    editInput.className = 'listContent';
+    editInput.style.backgroundColor = '#fffed3';
+    list.style.backgroundColor = '#fffed3';
+    editInput.value = listText.textContent;
+    list.replaceChild(editInput, listText);
+    editInput.addEventListener('keypress', (e) => {
+      if (e.key === 'Enter' && editInput.value) {
+        const getting = JSON.parse(localStorage.getItem('list'));
+        const result = getting.filter((word) => word.description === listText.textContent);
+        const empty = [];
+        for (let i = 0; i < getting.length; i += 1) {
+          if (getting[i].index === result[0].index) {
+            getting[i].description = editInput.value;
+          }
+          empty.push(getting[i]);
+          localStorage.setItem('list', JSON.stringify(empty));
+        }
+        list.replaceChild(listText, editInput);
+        listText.textContent = editInput.value;
+        list.style.backgroundColor = '#fff';
+      }
+    });
+  });
+};
 
-  displayListItem.appendChild(todoContainer);
+const addToLocalStorage = () => {
+  localStorage.setItem('list', JSON.stringify(array));
 };
 
-inputField.addEventListener('keypress', (e) => {
-  if (e.key === 'Enter' && inputField.value) {
+const dataEntry = document.querySelector('.dataEntry');
```

Comment:
- [ ] The new functionality should be on a new separate Javascript file, kindly create a new file and move all the new functionality's from `index.js` to the `new file.js`, this will make your code more organized and easy to maintain